common/server/statshouse/statshouse-client.cpp

```diff
@@ -0,0 +1,65 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2021 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include <fcntl.h>
+#include <unistd.h>
+
+#include "common/resolver.h"
+#include "net/net-connections.h"
+#include "runtime/critical_section.h"
+#include "statshouse-client.h"
```

Comment:
full path and move to top; the convention is:

#include "same file h"

#include <system>

#include "kphp files h"